src/tree/decorators.ts

```diff
@@ -0,0 +1,90 @@
+import {
+  Component,
+  DebugElement,
+} from '@angular/core';
```

Comment:
These imports were already in place, they were just in a separate file, `transform.ts`. I am not sure how to implement this code without at least `Component`.